client/client.go

```diff
@@ -25,10 +28,10 @@ func New(torrentPath string) (*Client, error) {
 		return nil, err
 	}
 	id := utils.ClientID()
-	var port uint16 = 6881
+	port := 6881
```

Comment:
should this be a flag rather than trying a bunch of ports?

Reply:
According to the [specs](https://www.bittorrent.org/beps/bep_0003.html#trackers), `Common behavior is for a downloader to try to listen on port 6881 and if that port is taken try 6882, then 6883, etc. and give up after 6889` so i set it that way. I should add a comment for that though.

Reply:
And now that it is an int, I should initialise it in the `for` loop.